api/internal/utils/utils.go

```diff
@@ -97,6 +98,16 @@ func InterfaceToString(val interface{}) string {
 	return str
 }
 
+func ObjectClone(origin, copy interface{}) error {
```

Comment:
We'd better to add some comments for this function, such as the object should not have float number, the `json.Marshal` and `json.Unmarshal` will cause the precision loss.

Reply:
added.